front/src/applications/stdcm/views/OSRDStdcmConfig.tsx

```diff
@@ -7,19 +7,13 @@ import { StdcmRequestStatus } from 'applications/stdcm/types';
 import STDCM_REQUEST_STATUS from 'applications/stdcm/consts';
 import ScenarioExplorer from 'common/ScenarioExplorer/ScenarioExplorer';
 import SpeedLimitByTagSelector from 'common/SpeedLimitByTagSelector/SpeedLimitByTagSelector';
-import {
-  getConf,
-  getInfraID,
-  getProjectID,
-  getScenarioID,
-  getStudyID,
-  getTimetableID,
-} from 'reducers/osrdconf/selectors';
 import { getSelectedTrain } from 'reducers/osrdsimulation/selectors';
 import { RollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector';
 import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import StdcmAllowances from 'modules/stdcmAllowances/components/StdcmAllowances';
 import { OsrdStdcmConfState } from 'applications/operationalStudies/consts';
+import { useOsrdSelectors } from 'common/osrdContext';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Comment:
Could you also update the other type imports of the file please
```suggestion
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Reply:
Done